develop/parachain-devs/interoperability/transfer-assets-with-xcm.md

```diff
@@ -0,0 +1,139 @@
+
+        ![](/images/develop/parachain-devs/interoperability/transfer-assets-with-xcm/transfer-assets-with-xcm-2.webp)
+
+3. Fill in the required parameters for the extrinsic
+    1. Enter the **dest** multilocation of the destination chain. In this case, since the destination chain is Asset Hub (ID `1000`), its Multilocation from the Relay Chain perspective is:
+        ```json
+        {
+            "parent": 0,
+            "interior": {
+                "X1": {
+                    "Parachain": 1000
+                }
+            }
+        }
+        ```
+    2. Specify the **beneficiary** to receive the assets. The beneficiary should be specified from the perspective of the destination chain. This tutorial uses the account `14BPeNxwQUBCDkZhAZMBTCD4ZbJ3VN4AdvDaUwDk8GxQbPbD` on the Asset Hub parachain, so the beneficiary MultiLocation is:
+        ```json
+        {
+            "parent": 0,
+            "interior": {
+                "X1": {
+                    "AccountId32": "14BPeNxwQUBCDkZhAZMBTCD4ZbJ3VN4AdvDaUwDk8GxQbPbD"
+                }
+            }
+        }
+        ```
+    3. Define the **assets** to teleport. This fields specifies the Multilocation fo the asset and the fungibility of the asset
+    4. Provide the **feeAssetItem** to define the asset to be used as the fee for the teleportation
+    5. Specify the **weightLimit** as the maximum weight to be used for the extrinsic execution
+    6. Click on **Submit Transaction** to execute the extrinsic
+
+        ![](/images/develop/parachain-devs/interoperability/transfer-assets-with-xcm/transfer-assets-with-xcm-3.webp)
+
+4. After submitting the transaction, you can check the status of the teleportation by navigating to the **Events** tab on the **Explorer** page for the relay chain in the Polkadot.js Apps interface. If the XCM message is successfully sent, you will see the event `xcmPallet.Sent` in the list of events
+
+    ![](/images/develop/parachain-devs/interoperability/transfer-assets-with-xcm/transfer-assets-with-xcm-4.webp)
+
+5. To verify the teleportation on the Asset Hub parachain, navigate to the **Events** section of the **Explorer** page. You should see two events, **`messageQueu.Processed`** and **`balances.Transfer`**
+
+    ![](/images/develop/parachain-devs/interoperability/transfer-assets-with-xcm/transfer-assets-with-xcm-5.webp)
+
+These events indicate that the teleportation of assets was successful. You can now verify the balance of the beneficiary account on the Asset Hub parachain to confirm the transfer.
+
+## Reserve Asset Transfer
+
+When consensus systems lack sufficient trust for direct asset teleportation, they can leverage a trusted third-party reserve (such as Asset Hub) to hold the actual assets. To track ownership of these reserve-held assets, both the source and destination chains typically create derivative tokens. Each chain maintains a dedicated account, known as a sovereign account, on the reserve to manage its asset holdings.
```

Comment:
Elaborated a bit more

```suggestion
It is a way of transferring assets between two parachains (or consensus systems) that don't trust each other by using a third system (like the Asset Hub system chain) they both trust, called the reserve. To track ownership of the reserve-held assets, both the source and destination chains typically create derivative tokens. Each chain maintains a dedicated account, known as a sovereign account, on the reserve to manage its asset holdings. 

Typically, the sender chain burns a certain amount of derivative tokens, signals the reserve to move the real assets from its sovereign account to the destination chain's sovereign account, and then signals the recipient to mint the right amount of derivatives.
```

Reply:
```suggestion
A reserve asset transfer transfers assets between two parachains (or consensus systems) that don't trust each other by using a third system (like the Asset Hub system chain) they both trust, called the reserve. Both the source and destination chains typically create derivative tokens to track ownership of the reserve-held assets. Each chain maintains a dedicated account, known as a sovereign account, on the reserve to manage its asset holdings. 
	
Typically, the sender chain burns a certain amount of derivative tokens, signals the reserve to move the real assets from its sovereign account to the destination chain's sovereign account, and then signals the recipient to mint the right amount of derivatives.
```

Grammarly improvements for the above suggestion